install-development-requirements.sh

```diff
@@ -25,5 +25,5 @@ pip install -r requirements-development.txt
 printf "\n"
 
 printf "=================== DOWNLOAD RENDER_TOOLS FROM GOLEM REPOSITORY ====================\n"
-./download_render_tools_from_golem_repo.sh
+./download_render_tools_from_golem_repository.sh
```

Comment:
Please use dashes in Bash script names: `download-render-tools-from-golem-repository.sh`.

Reply:
Done